tests/trilinos/nox_solver_01.with_trilinos=true.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::The solution is: 0.00390625
```

Comment:
This result is a bit counterintuitive, but one can nearly replicate the result on https://calculator-online.net/newtons-method-calculator/ so I suppose that its down to the chosen solver settings.